crypto/random/rand_utils.go

```diff
@@ -0,0 +1,51 @@
+package random
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gonum.org/v1/gonum/stat"
+)
+
+// BasicDistributionTest is a test function to run a basic statistic test on `randf` output.
```

Comment:
Randomness test tools consolidated here so that all `flow-go` tests can use them. 
Redundant code will be deleted once the new  crypto version is tagged.